shell/composables/useLabeledFormElement.ts

```diff
@@ -63,6 +64,10 @@ export const labeledFormElementProps = {
     type:    Boolean,
     default: false,
   },
+  requireDirty: {
+    default: true,
+    type:    Boolean
+  }
```

Comment:
This is a clear sign that there's some drift between the mixin and the composable. I created #11491 so that we can address.